Makefile

```diff
@@ -42,7 +41,8 @@ test:
 
 .PHONY: image
 image:
-	docker build -t $(REGISTRY)/$(IMAGE):$(TAG) . --target debian-base
+        # IMAGE will be overwritten by prow
+	docker build -t $(IMAGE) . --target debian-base
```

Comment:
It will be good we can rename this variable to something else if that's reused in the prow job, what I see is it is using some other infra image we are not suppose to use!

more info: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_ibm-powervs-block-csi-driver/129/pull-ibm-powervs-block-csi-driver-image-build/1497097897068466176

renaming `IMAGE` to `IMG` can be helpful

Reply:
fixed